MSCL/source/mscl/MicroStrain/Inertial/ExposedInertialTypes.h

```diff
@@ -2965,4 +2965,55 @@ namespace mscl
         // Trigger parameters
         EventTriggerParameters parameters;
     };
+
+    //API Struct: EventTriggerInfo
+    //  Information about an event trigger
+    struct EventTriggerInfo
+    {
+        //API Function: EventTriggerInfo
+        //  Constructor
+        EventTriggerInfo(const EventTriggerConfiguration::Trigger type, const uint8 status) :
+            type(type), status(status) {}
+
+        //API Enum: Status
+        //  Trigger status masks for the status bitfield
+        //      ACTIVE  - 0x00 - Trigger is active
+        //      ENABLED - 0x01 - Trigger is enabled
+        //      TEST    - 0x02 - Trigger is in test mode
+        enum Status
+        {
+            ACTIVE = 0x00, // Trigger is active
+            ENABLED = 0x01, // Trigger is enabled
+            TEST = 0x02  // Trigger is in test mode
+        };
+
+        //API Variable: type
+        //  Configured trigger type
+        EventTriggerConfiguration::Trigger type;
+
+        //API Function: getActiveStatus
+        //  Gets the status of the active bit from the status bitfield
+        uint8 getActiveStatus() const;
```

Comment:
imo it would be more intuitive for these to be bool isActive(), isEnabled() isTestMode() instead of returning a 1 or 0.
I'd pull the doc descriptions directly from the mip def bitfield definition table.